### PR TITLE
feat: handle erc1155 specs for nft transfer

### PR DESCRIPTION
**Pull request by bluecco**
erc721 transactions:
image

### CODE REVIEW

File: packages/extension/src/shared/call/__test__/erc721TransferCall.test.ts [hunk 1]
```diff
@@ -1,16 +1,16 @@
...
-  Erc721TransferCall,
-  parseErc721TransferCall,
-} from "../erc721TransferCall"
+  erc1155Transfer,
+  erc721Transfer,
+} from "../../../ui/features/accountActivity/transform/transaction/__test__/__fixtures__/transaction-calls/goerli-alpha"
+import { NftTransferCall, parseNftTransferCall } from "../nftTransferCall"
 
 describe("erc721TransferCall", () => {
   describe("parseErc721TransferCall()", () => {
```
Comment (simonheys) [inline packages/extension/src/shared/call/__test__/erc721TransferCall.test.ts:10]:
```suggestion
  describe("parseNftTransferCall()", () => {
```

File: packages/extension/src/shared/call/__test__/erc721TransferCall.test.ts [hunk 2]
```diff
@@ -22,4 +22,20 @@ describe("erc721TransferCall", () => {
       })
     })
   })
+
+  describe("parseErc721TransferCall()", () => {
```
Comment (simonheys) [inline packages/extension/src/shared/call/__test__/erc721TransferCall.test.ts:26]:
```suggestion
  describe("parseNftTransferCall()", () => {
```

File: packages/extension/src/shared/call/__test__/erc721TransferCall.test.ts (hunk 1 above)
Comment (simonheys) [reply packages/extension/src/shared/call/__test__/erc721TransferCall.test.ts:10]: Haha! I guess the file itself could be renamed too? Banish all the Erc721 specifics

Comment (bluecco) [reply packages/extension/src/shared/call/__test__/erc721TransferCall.test.ts:10]: def yeah

**Review by janek26 — APPROVED** [review]

**Pull request merged**